pydantic/main.py

```diff
@@ -169,6 +171,60 @@ def validate_custom_root_type(fields: Dict[str, Field]) -> None:
 TYPE_BLACKLIST = FunctionType, property, type, classmethod, staticmethod
 
 
+class ValueExclude:
```

Comment:
move out of `main.py`, maybe into `utils.py`

Reply:
also docstring explaining what this does too.

Reply:
Surely we want to use the same logic for include and exclude?

Reply:
All done in 927d0b9 :)